src/pages/getting_started/first_app.md

```diff
@@ -149,8 +149,21 @@ There are two sample flows listed below. Some developers may not have access to
     e2e			node_modules		package.json		web-src
     ```
     
-1. Note that you still can add/remove the extension points, back-end actions, SPA front-end or Github workflows from your application later by respectively using the `aio app <add|delete> ext`, `aio app <add|delete> action`, `aio app <add|delete> web-assets` and `aio app <add|delete> ci` commands within your application folder.
+7. Note that you still can add/remove the extension points, back-end actions, SPA front-end or Github workflows from your application later by respectively using the `aio app <add|delete> ext`, `aio app <add|delete> action`, `aio app <add|delete> web-assets` and `aio app <add|delete> ci` commands within your application folder.
 
+8. Optionally, you can install an App Builder template using Template Registry. Discover available templates in the App Builder template registry with the following command:
+
+    ```
+    $ aio templates discover
+    ```
+
+9. Install the desired template with the following command:
+
+    ```
+    $ aio templates install <npm package name>
+    ```
+    The template's npm package will be downloaded and extracted, and its installation configuration file parsed for workspace setup, APIs, event registration, and runtime required. `npm install` will run in the background to install the npm package.
```

Comment:
@AmyJZhao , [@adobe/aio-lib-console-project-installation](https://www.npmjs.com/package/@adobe/aio-lib-console-project-installation) doesn't parse the `event` section, only `workspaces`, `apis` and `runtime`, see https://github.com/adobe/aio-lib-console-project-installation